app.js

```diff
@@ -39,23 +44,17 @@ module.exports = (/*options*/) => {
   };
 
   // This function POSTs to Quay.
-  const authorizationSuccessful = (payload) => {
+  const authorizationSuccessful = (payload, url) => {
     console.log('Bitbucket has authenticated.');
     // Build a new payload that quay can consume.
     var quaypayload = {"commit":payload.changes[0].toHash,"ref":payload.changes[0].ref.id,"default_branch":payload.changes[0].ref.displayId};
-    // Make the POST request to the url in the config.
-    // Set different options if we're using our own CA.
-    if (process.env.QUAY_CA_FILE){
-      var quayoptions = {
-        json: quaypayload,
-        agentOptions: {
-          ca: fs.readFileSync(`/project/user-app/${process.env.QUAY_CA_FILE}`)
-        }
-      }
-    } else {
-      json: quayoptions
-    }
-    request.post(config[payload.repository.name], quayoptions, (error, res, body) => {
+    // Make the POST request to the request url.
+    instance.post('https://' + quayHost + req.originalUrl, quaypayload, {
+            auth: {
+                    username: '$token',
+                    password: req.query.token
```

Comment:
If we do not want to use the `req` obj, we also need to pass a `token` var. See comment above.